jbpm-wb-showcase/src/main/java/org/jbpm/workbench/client/ShowcaseEntryPoint.java

```diff
@@ -57,11 +58,14 @@
 
     protected WorkbenchMegaMenuPresenter menuBar;
 
+    protected DefaultAdminPageHelper adminPageHelper;
```

Comment:
Have just one question regarding introduction of this object. Why it's needed? I don't see it's used anywhere.